README.md

```diff
@@ -401,7 +401,7 @@ export class LoggerPlugin implements NgxsPlugin {
 }
 ```
 
-This pattern allows us to define options while presevering the constructor
+This pattern allows us to define options while persevering the constructor
```

Comment:
I think this should be 'preserving'

Reply:
Oh, I see what you're saying

Reply:
Reverted back to the original line and amended the commit.